webclient/app/src/app/commons/alert/ConfirmationDialog.jsx

```diff
@@ -1,32 +1,60 @@
 import * as React from 'react';
+import {useState} from 'react';
 import Button from '@mui/material/Button';
 import Dialog from '@mui/material/Dialog';
 import DialogActions from '@mui/material/DialogActions';
-import Alert from '@mui/material/Alert';
-import AlertTitle from '@mui/material/AlertTitle';
-import { useTranslation } from 'react-i18next';
+import {useTranslation} from 'react-i18next';
+import {DialogContent, DialogContentText, DialogTitle} from "@mui/material";
+import PropTypes from "prop-types";
+import {LoadingButton} from "@mui/lab";
 
 export default function ConfirmationDialog(props) {
-    const { content, open, onClose, onSubmit, title, message } = props;
-    const { t } = useTranslation();
+    const {open, onClose, onSubmit, title, message, confirmTitle} = props;
+    const {t} = useTranslation();
+    const [isProcessing, setIsProcessing] = useState(false);
+
+    function confirmAction() {
+        setIsProcessing(true);
+        onSubmit()
+            .then(() => {
+                onClose();
+                setIsProcessing(false);
+            })
+            .catch(() => {
+                setIsProcessing(false)
+            })
+    }
 
     return (
         <Dialog
             open={open}
             onClose={onClose}
-            aria-labelledby="alert-dialog-title"
-            aria-describedby="alert-dialog-description"
+            aria-labelledby="confirm-dialog-title"
+            aria-describedby="confirm-dialog-description"
         >
-            <Alert severity="warning">
-                <AlertTitle>{content ? content.title : title}</AlertTitle>
-                {content ? content.message : message}
-            </Alert>
+            <DialogTitle id="confirm-dialog-title">
+                {title}
+            </DialogTitle>
+            <DialogContent>
+                <DialogContentText id="confirm-dialog-description">
+                    {message}
+                </DialogContentText>
+            </DialogContent>
             <DialogActions>
-                <Button onClick={onClose}>{t("button.no")}</Button>
-                <Button onClick={onSubmit} autoFocus>
-                    {t("button.yes")}
-                </Button>
+                <Button onClick={onClose}>{t("button.cancel")}</Button>
```

Comment:
Did you test the cancel button? It was not working on my side in the old code.

Reply:
Works on my side